src/uu/stdbuf/src/libstdbuf/src/libstdbuf.rs

```diff
@@ -38,10 +39,10 @@ fn set_buffer(stream: *mut FILE, value: &str) {
         "0" => (_IONBF, 0_usize),
         "L" => (_IOLBF, 0_usize),
         input => {
-            let buff_size: usize = match input.parse() {
-                Ok(num) => num,
-                Err(e) => crash!(1, "incorrect size of buffer!: {}", e),
-            };
+            let buff_size: usize = input
+                .parse()
+                .map_err(|e| format!("incorrect size of buffer!: {}", e))
+                .unwrap();
```

Comment:
`unwrap` will still panic, so I don't think that's much better than `crash!`.

Reply:
You're right, I'll wait to see how this should be handled before fixing.